Formula/qemu.rb

```diff
@@ -36,18 +36,76 @@ class Qemu < Formula
   depends_on "vde"
 
   on_linux do
+    depends_on "attr"
     depends_on "gcc"
     depends_on "gtk+3"
+    depends_on "libcap-pg"
```

Comment:
libcap-ng ?

Reply:
See the message above

Reply:
Oh nevermind, I reread it now.
```suggestion
    depends_on "libcap-ng"
```